### DISCUSSION TITLE
Who to contact for security issues

### DISCUSSION THREAD

**Discussion by JamieSlome** [opencrx/opencrx#118] [Q&A]
Hello 👋

I run a security community that finds and fixes vulnerabilities in OSS. A researcher (@ciph0x01) has found a potential issue, which I would be eager to share with you.

Could you add a `SECURITY.md` file with an e-mail address for me to send further details to? GitHub [recommends](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) a security policy to ensure issues are responsibly disclosed, and it would help direct researchers in the future.

Looking forward to hearing from you 👍

(cc @huntr-helper)

**Comment by christoph-mueller-crixp** [comment]
just uploaded `SECURITY.md`
please submit your security/vulnerability report directly to security@opencrx.org
Thx, Chris
